src/main/java/com/squareup/javapoet/AnnotationSpec.java

```diff
@@ -226,6 +228,26 @@ public Builder addMember(String name, CodeBlock codeBlock) {
       return this;
     }
 
+    public Builder addMember(String name, Object value) {
```

Comment:
Is changing this method signature to `addValue(String memberName, Object value)` more precise?
 * it's more distant to existing `addMember(String name, String format, Object... args)`
 * it's closer to the new `addValue()` with implicite `memberName="value"`